biorbd_optim/constraints.py

```diff
@@ -132,6 +132,7 @@ def continuity(ocp):
         for i in range(len(ocp.nlp) - 1):
             if ocp.nlp[i]["nx"] != ocp.nlp[i + 1]["nx"]:
                 raise RuntimeError("Phase constraints without same nx is not supported yet")
+        for i in range(len(ocp.phase_transitions)):
```

Comment:
Can you make this a proper for loop (for phase in ocp.phase_transitions) instead of range(len()) :)